.gitignore

```diff
@@ -19,11 +19,15 @@ pom.xml.releaseBackup
 release.properties
 
 autom4te.cache/
+src/.vs
+src/**/__history
```

Comment:
I have directories like the following as well in my tree:

> src/**/__recovery

Not sure if those are C++-Builder specific, as I use the same tree for Visual Studio and C++-Builder.

Reply:
They may be; I pulled this from one of your branches, maybe it never got added to the gitignore?  It's not hard to add though.

Reply:
If you pulled from me, than they are C++-Builder specific and you should add `__recovery` as well. I do have some `__astcache`-dirs as well, but not in the source tree. So let's focus on what is really needed. Don't know myself when `__recovery` gets created, though.